deeplabcut/pose_estimation_tensorflow/datasets/pose_multianimal_imgaug.py

```diff
@@ -97,6 +121,53 @@ def load_dataset(self):
         self.has_gt = has_gt
         return data
 
+    @lru_cache(maxsize=None)
+    def read_image_shape_fast(self, path):
+        # Blazing fast and does not load the image into memory
+        with Image.open(path) as img:
+            width, height = img.size
+            return len(img.getbands()), height, width
+
+    def _load_pseudo_data_from_h5(self, cfg, threshold=0.5):
```

Comment:
`threshold` is not used; where should it go?

Reply:
Right let me fix it

Reply:
Thanks! Fixed